examples/screen_cast_pw.rs

```diff
@@ -0,0 +1,218 @@
+use std::os::fd::{IntoRawFd, OwnedFd};
+
+use ashpd::desktop::{
+    screencast::{CursorMode, Screencast, SourceType, Stream as AshStream},
+    PersistMode,
+};
+use pipewire as pw;
+use pw::{properties::properties, spa};
+
+use spa::pod::Pod;
```

Comment:
```suggestion
use pw::{properties::properties, spa::{self, pod::Pod}};
```

Reply:
just getting rid of this distinction since Pod only had a single reference  anyway; qualifying it as spa::pod::Pod now.